examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -173,29 +200,18 @@ function implicit_tendency_special!(Yₜ, Y, p, t)
     ᶜρ = Y.c.ρ
     ᶜuₕ = Y.c.uₕ
     ᶠw = Y.f.w
-    (; ᶠgradᵥ_ᶜΦ, params, ᶠupwind_product) = p
-    thermo_params = CAP.thermodynamics_params(params)
-    # Used for automatically computing the Jacobian ∂Yₜ/∂Y. Currently requires
-    # allocation because the cache is stored separately from Y, which means that
-    # similar(Y, <:Dual) doesn't allocate an appropriate cache for computing Yₜ.
     (; ᶜK, ᶜts, ᶜp) = implicit_cache_vars(Y, p)
+    (; ᶠgradᵥ_ᶜΦ, params, energy_upwinding, tracer_upwinding, simulation) = p
+    thermo_params = CAP.thermodynamics_params(params)
+    dt = simulation.dt
 
     ref_thermo_params = Ref(thermo_params)
     ref_zuₕ = Ref(zero(eltype(Yₜ.c.uₕ)))
 
     @nvtx "implicit tendency special" color = colorant"yellow" begin
         Fields.bycolumn(axes(Y.c)) do colidx
-
-            if p.tendency_knobs.has_turbconv
```

Comment:
Is this change unrelated to this PR? Otherwise I am not sure I follow entirely why this would change with vertical upwinding mode

Reply:
Yeah this was just me doing some housekeeping to make the special and generic tendencies look identical, since I was already modifying them.